cn/docs/code/quick_start/lenet_eval.py

```diff
@@ -76,8 +76,11 @@ def acc(labels, logits):
 
 if __name__ == "__main__":
 
-    check_point = flow.train.CheckPoint()
-    check_point.load("./lenet_models_1")
+    if flow.__version__ < "0.3.0":
```

Comment:
Comparing versions as string seems cause problems. eg: `"0.25.0" < "0.3.0"` evaluted to `True`, besides, versions can be `0.1b.0` formats.

You can just remove the old api call, and use latest one only, because our tutorial has a corresponding version to oneflow(see docs.oneflow.org right down corner).

Thank you for your contribution, this pr will be merged soon after modification talked about above.

Reply:
是的。我再改下。谢谢。

Reply:
辛苦，另外我们的en目录下，也有一套几乎一样的代码（区别可能只有注释是英文的），你可以帮忙一起改了吗 :-D

Reply:
好点。已改。